static/js/components/forms/B2BPurchaseForm.js

```diff
@@ -24,7 +24,8 @@ type Props = {
   fetchCouponStatus: (payload: B2BCouponStatusPayload) => Promise<*>,
   contractNumber: ?string,
   discountCode: ?string,
-  productId: ?string
+  productId: ?string,
```

Comment:
This does not appear to be based on `master`. Please rebase so you can work with the latest code base which has product as an object rather than a string.

Reply:
yes, I know and going to rebase it. It has conflicts with the master branch.